skills/csharp/tests/musicskill.Tests/ControlSettingsDialogTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.AI.Luis;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Solutions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MusicSkill.Models;
+using MusicSkill.Responses.ControlSettings;
+using MusicSkill.Responses.Main;
+using MusicSkill.Services;
+using MusicSkill.Tests.Utilities;
+using MusicSkill.Tests.Utterances;
+
+namespace MusicSkill.Tests
+{
+    [TestClass]
+    public class ControlSettingsDialogTests : SkillTestBase
+    {
+        [TestInitialize]
+        public void SetupLuisService()
+        {
+            var botServices = Services.BuildServiceProvider().GetService<BotServices>();
+            botServices.CognitiveModelSets.Add("en-us", new CognitiveModelSet()
+            {
+                LuisServices = new Dictionary<string, LuisRecognizer>()
+                {
+                    { "General", GeneralTestUtil.CreateRecognizer() },
+                    { "MusicSkill", ControlSettingsTestUtil.CreateRecognizer() }
+                }
+            });
+        }
+
+        [TestMethod]
+        public async Task Test_Pause()
+        {
+            await GetTestFlow()
+                .Send(string.Empty)
```

Comment:
use SendConversationUpdate()

Reply:
Most skill use string.Empty, so we can align it later?